firefox-ios/firefox-ios-tests/Tests/StorageTests/RustLoginsTests.swift

```diff
@@ -8,10 +8,120 @@ import XCTest
 
 @testable import Storage
 
+// This class uses the logic enabled with the `useRustKeychain` feature flag. It will replace
```

Comment:
@issammani Let me know if this is what you had in mind. My thinking is `LegacyRustLoginsTests` gets removed with the nimbus code.

Reply:
Yes this makes sense. Thanks for adding it.